src/main/webapp/app/admin/user-management/user-management-update.component.scss

```diff
@@ -28,3 +28,9 @@
         color: var(--text-editor-color);
     }
 }
+
+::ng-deep .group-panel {
+    background-color: var(--text-editor-background);
+    color: var(--text-editor-color);
+    border: 1px solid gray;
```

Comment:
Please do not use hardcoded colors. Use default colors. Refer to https://docs.artemis.cit.tum.de/dev/guidelines/client-design/ for more information on colors

Reply:
done